### PR TITLE
Added 6 temp views, added navigation logic

### PR DESCRIPTION
**Pull request by JonathanPartain**

### CODE REVIEW

File: src/views/PriceSuggestion/Bestallning.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import {
+    View,
+    StyleSheet,
+    Text,
+} from 'react-native';
+
+export default class Bestallning extends Component {
```
Comment (jeremydanner) [inline src/views/PriceSuggestion/Bestallning.js]: Fix class Name?

File: src/views/PriceSuggestion/Hustyp.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import {
+    View,
+    StyleSheet,
+    Text,
+} from 'react-native';
+
+export default class Hustyp extends Component {
```
Comment (jeremydanner) [inline src/views/PriceSuggestion/Hustyp.js]: Fix class name to English

File: src/views/PriceSuggestion/Kundupgifter.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import {
+    View,
+    StyleSheet,
+    Text,
+} from 'react-native';
+
+export default class Kundupgifter extends Component {
```
Comment (jeremydanner) [inline src/views/PriceSuggestion/Kundupgifter.js]: Class name

**Review by jeremydanner — CHANGES_REQUESTED** [review]

File: src/views/PriceSuggestion/PriceSuggestion.js [hunk 4]
```diff
@@ -1,27 +1,181 @@
...
+        this.state = {
+            activeTab: <Skorstenstyp />,
+            currentTab: "skorstenstyp",
+
+        }
+
+    }
+
+    // functions to set active tab to set tab file
+    skorstenstyp(){
```
Comment (jeremydanner) [inline src/views/PriceSuggestion/PriceSuggestion.js]: Change the method names to English!

**Review by jeremydanner — CHANGES_REQUESTED** [review]

**Comment by JonathanPartain** [comment]
@jeremydanner Please make sure everything looks as it should, then merge!

**Review by jeremydanner — APPROVED** [review]
Topp! Jag mergar.

**Pull request merged**